packages/components/datepicker/src/Calendar/Calendar.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+import { getStyles } from './Calendar.styles';
+
+import { DayPicker, DayPickerProps } from 'react-day-picker';
+import { CalendarCaption } from './CalendarCaption';
+import { CalendarCaptionLabel } from './CalendarCaptionLabel';
+
+// Props comparison to our current datepickers:
+// Current      DayPicker
+// value        selected
+// onChange     onSelect
+// min          fromDate/fromMonth/fromYear
+// max          toDate/toMonth/toYear
+
+export type CalendarProps = DayPickerProps;
+
+export function Calendar(props: CalendarProps) {
```

Comment:
Can we use jsdoc here? some info with a link to DayPickerProps docs will be cool